com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/HeliumEventProcessor.kt

```diff
@@ -0,0 +1,76 @@
+package com.chartboost.heliumsdk.unity
+
+import android.util.Log
+import com.chartboost.heliumsdk.ad.HeliumAdError
+import org.json.JSONException
+import org.json.JSONObject
+
+object HeliumEventProcessor {
+    private const val TAG = "HeliumEventProcessor"
+
+    @JvmStatic
+    fun serializeHeliumEvent(
+        placementName: String,
+        heliumAdError: HeliumAdError?,
+        eventConsumer: HeliumEventConsumer<String, Int, String>
+    ) {
+        eventConsumer.accept(placementName, heliumAdError?.getCode() ?: -1, heliumAdError?.getMessage() ?: "")
+    }
+
+    @JvmStatic
+    fun serializeHeliumBidEvent(
+        placementName: String,
+        dataMap: HashMap<String, String>,
+        eventConsumer: HeliumBidEventConsumer<String, String, String, Double>
+    ) {
+        val partnerId = dataMap["partner_id"] ?: ""
+        val auctionId = dataMap["auction-id"] ?: ""
+        val price = try {
+            dataMap["price"]?.toDouble() ?: 0.0
+        } catch (e: NumberFormatException) {
+            Log.d(TAG, "HeliumBidEvent failed to serialize price, defaulting", e)
```

Comment:
I get that it's defaulting to 0, but in the log message we should say what it defaults to.

Reply:
I did consider it, but I guess I forgot about it :P